.github/workflows/ci.yml

```diff
@@ -67,6 +67,7 @@ jobs:
       - name: Install dependencies
         run: |
           pip3 install -r requirements.txt
+          pip3 install networkx
```

Comment:
Question: any reason to install here instead of within `test/requirements.txt`? Does installing from `requirements.txt` automatically install with `--pre`?

Reply:
`requirements.txt` does not contain `networkx`. This is a dependency coming from `torch`. And since we are installing that with the `--pre` flag for nightlies, we also pull in a pre-release of the other dependencies. By installing it explicitly before, the dependency is already satisfied and `pip` does not install the pre-release versions.